test/ripemd160Test.mo

```diff
@@ -0,0 +1,86 @@
+import Array "mo:base/Array";
+import Blob "mo:base/Blob";
+import Iter "mo:base/Iter";
+import Text "mo:base/Text";
+import Debug "mo:base/Debug";
+import Ripemd160 "../src/Ripemd160";
+
+let testData: [(Text, [Nat8])] = [
+  (
+    "",
+    [156, 17, 133, 165, 197, 233, 252, 84, 97, 40, 8, 151, 126, 232, 245, 72, 178, 37, 141, 49]
```

Comment:
nit: These test vectors would be much easier for a reader to verify if they were in hex.

Reply:
True, changed.